external/test-runtime/optional.csproj

```diff
@@ -0,0 +1,12 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="..\..\dependencies.props" />
+  <PropertyGroup>
+    <TargetFramework>net45</TargetFramework>
```

Comment:
For the testILC stuff they're going to be needed when the target framework is uapaot, though if the optional project doesn't specify this I'd expect it to work.  We can test this on Monday.

Reply:
I haven't worked with this tooling before.  But from what I can tell, the reason it is using the `net45` TFM is because that's where the `IBCMerge.exe` comes from - the `net45` assets.  See https://github.com/dotnet/buildtools/blob/03988127f489bcd1cf58d71131ac628620d53de9/src/Microsoft.DotNet.Build.Tasks/PackageFiles/OptionalTooling.targets#L124-L131 where it is pulling all the references out (notice the .NET Framework, v4.5 hardcoded there too).

For the testILC stuff, it appears those packages just need to be restored/downloaded.  There is code here https://github.com/dotnet/corefx/blob/84883052ef0f720a8befdc8525db50c9544227d5/external/test-runtime/XUnit.Runtime.depproj#L196-L201 that reads it directly from the `$(PackagesDir)`, and doesn't care about the TFM.